src/probes/telem_record_gen.c

```diff
@@ -91,7 +94,7 @@ int parse_options(int argc, char **argv)
         long unsigned int tmp = 0;
 
         int opt;
-        while ((opt = getopt_long(argc, argv, "hc:Vs:c:p:P:R:", prog_opts, NULL)) != -1) {
+        while ((opt = getopt_long(argc, argv, "hc:Vs:c:p:P:R:e:", prog_opts, NULL)) != -1) {
```

Comment:
If the -e option is indeed optional, then you'll want to specify `e::` (two colons). If it is required, then `e:` (as-is) looks good.